lms/templates/courseware/accordion.html

```diff
@@ -17,7 +17,7 @@
       %>
       <h3 ${active_class} aria-label="${aria_label}">
         <a href="#">
-          ${chapter['display_name']}
+            ${chapter['chapters_number']}.  ${chapter['display_name']}
```

Comment:
Why showing 'chapters_number'? I believe this is not the part of bug you are fixing. 
Create a separate PR for this change.